Fluent/Physics.ipynb

```diff
@@ -0,0 +1,136 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# CFD Problem's Physics & Theory\n",
+    "\n",
+    "**In developing CFD simulation for a fluid mechanic problem understanding the general physics and fundamental theory of the problem is extremely important. This knowledge would provide users the ability to make logical decision about the geometry of CFD domain, choice of proper boundary conditions and numerical models for the CFD simulation. Furthermore, user will have a general big picture on the expected outcome of simulation. These knowledges and abilities form the foundation for a successful implementation and validation of the CFD simulation.**\n",
+    "\n",
+    "**In other words understanding physics and theory of problem before developing a CFD simulation can be thought of turning on a flash light before taking any step in a completely dark room. Hence, let's review the fundamental physics and theory behind the problem of \"[insert the topic of problem here]\" as the first step for development and validation of CFD simulation for this problem of interest:**\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Control volume analysis - conservation of mass\n",
+    "\n",
+    "One of the key features for the flow over a cylinder is the pattern of the flow. In other words, we need to determine the *stream function* of the flow. To obtain the stream function of the flow, we need to apply Control Volume (CV) analysis to a certain area around the cylinder and derive from the conservation of mass.\n",
```

Comment:
The explanation approach is little confusing to a potential reader. Revise the structure according to following guideline by changing the position of currently written paragraphs and modifying the content from linguistic point of view:

1-Explain the general physics of the flow flied in simple words and using a simple sketch maybe.
2-Briefly discuss that you use CV analysis to estimate the velocity field and forces on the blunt object (i.e. Cylinder).
3-Start with conservation of mass and state the assumptions that would simplify the equation.
! Please remember considering 2D flow and neglecting the velocity variation in y-direction, conservation of mass can estimate velocity field. There is no need for stream function (i.e. potential flow theory). So take that part out.
4-Briefly discuss how forces, specifically drag force on cylinder, is important variable and conservation of momentum will give us that.
5-Present general form of conservation of momentum equation. State assumptions and how the equation will simplify to give you drag force.
6-Talk about importance of normalizing the drag force and define drag coefficient (you can copy-paste the explanation provided in Simulation and Post-Processing sections).
7-Discuss how CFD will give a more accurate answer and is capable of solving this problem with less simplification and assumptions.